src/Movim/Template/Builder.php

```diff
@@ -138,55 +139,31 @@ public function title(): string
      */
     public function language(): string
     {
-        $lang = $this->user && $this->user->language
-            ? $this->user->language
-            : Configuration::get()->locale;
-
-        if (empty($lang)) {
-            return 'en';
+        if ($this?->user?->language != null) {
+            $this->lang = $this->user->language;
+            return \Movim\i18n\Locale::printISO639($this->lang);
```

Comment:
Same there, its useful when it's used only once, but if the Class is used several times better import it once.